lms/static/js/learner_dashboard/views/course_enroll_view.js

```diff
@@ -0,0 +1,44 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone',
+            'jquery',
+            'underscore',
+            'gettext',
+            'edx-ui-toolkit/js/utils/html-utils',
+            'text!../../../templates/learner_dashboard/course_enroll.underscore'
+           ],
+         function(
+             Backbone,
+             $,
+             _,
+             gettext,
+             HtmlUtils,
+             pageTpl
+         ) {
+            return Backbone.View.extend({
+                tpl: HtmlUtils.template(pageTpl),
+
+                events: {
+                    'click .enroll-button': 'handleEnroll'
+                },
+
+                initialize: function(options) {
+                    if (options.$el){
+                        this.$el = options.$el;
+                    }
+                    this.render();
```

Comment:
It would make sense to move this inside of the if loop, otherwise render will throw an error if options.$el is undefined.